### PR TITLE
Adding file and secretsmanager plugins and fixed a few log messages

### PR DESCRIPTION
**Pull request by kcbraunschweig**
- fixed a few log messages and comments noticed in testing
- created file plugin when you want to place a file without having to use bundle + some shell scripting
- created secretsmanager plugin to use as a source a secret string stored in AWS secretsmanager service
these 2 plugins are handy when used together.

### CODE REVIEW

**Comment by kcbraunschweig** [comment]
open to input on what that windows failure is. doesn't look to me like anything i changed and i have no ability to test on windows.

File: plugin/source/secretsmanager/secretsmanager.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+	}
+	outpath := filepath.Join(dlPath, s.FileName)
+
+	result, err := svc.GetSecretValue(input)
+	if err != nil {
+		s.logger.Debugf(0, "failed to retrieve secret %s: %s", s.SecretId, err)
+		return err
+	}
+
+	// Create the otuput file if it doesn't exist
```
Comment (svmastersamurai) [inline plugin/source/secretsmanager/secretsmanager.go]: typo
```suggestion
	// Create the output file if it doesn't exist
```

File: plugin/source/secretsmanager/secretsmanager.go [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+func (s *Source) Type() string {
+	return TypeName
+}
+
+// SetName sets the name of this source instance
+func (s *Source) SetName(name string) {
+	s.SourceName = name
+}
+
+// DownloadToPath performs reads the SecretString from secretsmanager and
```
Comment (svmastersamurai) [inline plugin/source/secretsmanager/secretsmanager.go]: You can drop this word and it reads better.
```suggestion
// DownloadToPath reads the SecretString from secretsmanager and
```

File: plugin/step/file/file.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
+package file
+
+/*
+	Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+*/
+
+import (
+	//	"os"
```
Comment (svmastersamurai) [inline plugin/step/file/file.go]: If you don't need it you can remove it.
```suggestion
```

**Comment by svmastersamurai** [comment]
a;so the build failure is a known issue right now, I wouldn't worry about it as long as the changes still build + test okay on non-windows platforms.

**Pull request merged**